src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -19,15 +28,38 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
+        List<Predicate<Person>> listOfPredicates = new ArrayList<>();
```

Comment:
```suggestion
        List<Predicate<Person>> predicates = new ArrayList<>();
```

Could consider renaming it to predicates?